ui/app/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -251,6 +257,37 @@ export default class NetworkForm extends PureComponent {
     this.setErrorTo('chainId', errorMessage)
   }
 
+  validateChainIdOnSubmit = async (chainId, rpcUrl) => {
+    const { t } = this.context
+    let errorMessage
+    let endpointChainId
+    let providerError
+
+    try {
+      const provider = new ethers.providers.JsonRpcProvider(rpcUrl)
+      endpointChainId = await provider.send('eth_chainId')
+    } catch (err) {
+      log.error(err)
+      providerError = err
+    }
+
+    if (providerError || typeof endpointChainId !== 'string') {
+      errorMessage = t('failedToFetchEndpointChainId')
```

Comment:
Wrong message key here:
```suggestion
      errorMessage = t('failedToFetchChainId')
```

Reply:
911491d